audit/projects/k8s-staging-artifact-promoter/services/enabled.txt

```diff
@@ -1,12 +1,10 @@
 NAME                              TITLE
 cloudbuild.googleapis.com         Cloud Build API
 cloudkms.googleapis.com           Cloud Key Management Service (KMS) API
-compute.googleapis.com            Compute Engine API
```

Comment:
Service changes in k8s-staging projects are expected due to https://github.com/kubernetes/k8s.io/pull/1859#issuecomment-814444712

This addresses https://github.com/kubernetes/k8s.io/issues/1675